FastAndFurious/FFRaceTrackViewController.m

```diff
@@ -89,6 +89,7 @@ - (NSString *)idForWinner {
         float finishLineXValue = self.view.frame.size.width - view.frame.size.width;
         if (x >= finishLineXValue && x >= furthestX) {
             idOfFurthest = identifier;
+            furthestX = x;
```

Comment:
no, we only keep the furthest distance so that the identifier will update only if they surpass that distance